Source/DataSources/PolylineGraphics.js

```diff
@@ -30,6 +30,7 @@ define([
      * @param {Property} [options.width=1.0] A numeric Property specifying the width in pixels.
      * @param {Property} [options.show=true] A boolean Property specifying the visibility of the polyline.
      * @param {MaterialProperty} [options.material=Color.WHITE] A Property specifying the material used to draw the polyline.
+     * @param {MaterialProperty} [options.depthFailMaterial=undefinded] A property specifiying the material to used to draw the polyline when it is below the terrain.
```

Comment:
We usually just do `[options.depthFailMaterial]` instead of `[options.depthFailMaterial=undefinded]`